### PR TITLE
Update material-ui.d.ts

### PR DESCRIPTION
**Pull request by kirill-chaban**
Added missing 'style' property for DatePicker

### CODE REVIEW

File: material-ui/material-ui.d.ts [hunk 1]
```diff
@@ -317,6 +317,7 @@ declare namespace __MaterialUI {
             onTouchTap?: React.TouchEventHandler;
             shouldDisableDate?: (day: Date) => boolean;
             showYearSelector?: boolean;
+			style?: React.CSSProperties;
```
Comment (adidahiya) [inline material-ui/material-ui.d.ts]: use spaces like the rest of the file, not tabs

**Comment by kirill-chaban** [comment]
Fixed according to the comment

File: material-ui/material-ui-tests.tsx [hunk 2]
```diff
@@ -162,7 +163,7 @@ class MaterialUiTests extends React.Component<{}, {}> implements React.LinkedSta
             </Card>;
 
         // "http://material-ui.com/#/components/date-picker"
-
+		<DatePicker style={{ color: 'red' }} />;
```
Comment (adidahiya) [inline material-ui/material-ui-tests.tsx]: you are still using tabs here

**Comment by kirill-chaban** [comment]
Can you please check this small change?

**Comment by vvakame** [comment]
I decided to merge this PR. thanks mate.

**Pull request merged**